provider/go.mod

```diff
@@ -24,3 +17,74 @@ require (
 	google.golang.org/api v0.46.0
 	google.golang.org/grpc v1.37.0
 )
+
+require (
```

Comment:
Does this need a `go mod tidy` run on it or is this normal?

Reply:
Ignore that ... just reading up on the background looks like this is normal.

Reply:
This is normal for go 1.17 modules. All of the transitive deps are explicit in the `go.mod` now.